pkgs/games/sfrotz/default.nix

```diff
@@ -0,0 +1,72 @@
+{
+  fetchFromGitLab,
+  freetype,
+  libao,
+  libjpeg,
+  libmodplug,
+  libpng,
+  libsamplerate,
+  libsndfile,
+  libvorbis,
+  pkg-config,
+  SDL2,
+  SDL2_mixer,
+  stdenv,
+  zlib
+}:
```

Comment:
The idiomatic style for long argsets is leading comma, not trailing. :-)

Reply:
Thanks, changed this.